### PR TITLE
small update of an offline test

### PR DESCRIPTION
**Pull request by LenkaNovak**
Small fix to update an offline test for the GCM spectrum diagnostic.

### Description

- [x] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code is exercised in an integration test OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

**Review by kpamnany — APPROVED** [review]

File: test/Common/Spectra/gcm_standalone_visual_test.jl [hunk 1]
```diff
@@ -91,8 +95,13 @@ contourf(
...
 # Check magnitude
-println(sum(spectrum))
-println(sum(
-    0.5 .* var_grid[:, :, 1] .^ 2 *
-    reshape(sqrt.(1 .- sinθ .^ 2), (length(sinθ), 1)),
-))
+println(0.5 .* sum(spectrum))
+
+dθ = π .* wts ./ sum(wts)
+dθ = π / length(wts)
```
Comment (jakebolewski) [inline test/Common/Spectra/gcm_standalone_visual_test.jl:101]: this seems to be overriding the same variable value?

Comment (kpamnany) [reply test/Common/Spectra/gcm_standalone_visual_test.jl:101]: An F for my review! :(

Comment (LenkaNovak) [reply test/Common/Spectra/gcm_standalone_visual_test.jl:100]: Ahh well spotted, thanks! `wts` is 1 anyway, so either should work. All fixed! :)

**Review by jiahe23 — APPROVED** [review]

**Pull request merged**